packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/map_configuration.dart

```diff
@@ -44,6 +46,16 @@ class MapConfiguration {
   /// See [WebGestureHandling] for more details.
   final WebGestureHandling? webGestureHandling;
 
+  /// This setting controls how the API handles cameraControl button position on the map. Web only.
+  ///
+  /// See [WebCameraControlPosition] for more details.
```

Comment:
Same comment here; the null behavior should be documented.

Reply:
done.